_install-and-configure/configuring-opensearch/network-settings.md

```diff
@@ -37,6 +37,8 @@ OpenSearch supports the following advanced network settings for HTTP communicati
 
 - `http.publish_host` (Static, list): Specifies an address or addresses that an OpenSearch node publishes to other nodes for HTTP communication.
 
+- `http.compression`. (Static, Boolean) Enables support for compression when applicable (using Accept-Encoding). When HTTPS is enabled, the default is set to `false`; otherwise, it defaults to `true`. Disabling compression for HTTPS helps mitigate potential security risks, such as BREACH attacks. To enable compression for HTTPS traffic, you must explicitly set `http.compression: true`.
```

Comment:
```suggestion
- `http.compression`. (Static, Boolean) Enables support for compression using `Accept-Encoding` when applicable. When `HTTPS` is enabled, the default is `false`. otherwise the defaults is `true`. Disabling compression for HTTPS helps mitigate potential security risks, such as `BREACH` attacks. To enable compression for HTTPS traffic, explicitly set `http.compression:` to `true`.
```